bindings/matrix-sdk-ffi/CHANGELOG.md

```diff
@@ -2,10 +2,14 @@
 
 Breaking changes:
 
-- `EventSendState` now has two additional variants: `VerifiedUserHasUnsignedDevice` and
-  `VerifiedUserChangedIdentity`. These reflect problems with verified users in the room
-  and as such can only be returned when the room key recipient strategy has
-  `error_on_verified_user_problem` set.
+- `EventSendState` now has four additional variants: `VerifiedUserHasUnsignedDevice`,
+  `VerifiedUserChangedIdentity`, `CrossSigningNotSetup`, and
+  `SendingFromUnverifiedDevice`. The first two reflect problems with verified users in
+  the room and as such can only be returned when the room key recipient strategy has
+  `error_on_verified_user_problem` set, or when using the identity-based strategy. The
+  last two indicate that your own device is not properly cross-signed, which is a
+  requirement when using the identity-based strategy, and can only be returned when
+  using the identity-based strategy.
```

Comment:
erm, we need to make new changelog entries, not just modify old changelogs that apply to old versions that are already built into applications.

Reply:
OK, I wasn't sure about this, because it's under the "unreleased" section, so wasn't sure if it was better to modify the existing entry, or have two separate entries that are related to the same area.

Reply:
unfortunately everything is "unreleased" at the moment :(